tpr/tpr.go

```diff
@@ -51,13 +49,14 @@ type Config struct {
 type TPR struct {
 	clientset kubernetes.Interface
 
-	name          string
-	group         string
-	version       string
-	description   string
-	qualifiedName string // name.group
+	name        string // see Config.Name
```

Comment:
Also, is the `name` field needed anymore, or is it superseded by `kind` and `group`?